master/buildbot/steps/artifact.py

```diff
@@ -298,6 +362,13 @@ def rsyncWithRetry(step, origin, destination, port=None):
 
     return retryCommandLinuxOS(rsync_command)
 
+def mkDir(step):
+    if _isWindowsSlave(step):
+        return r'C:\cygwin64\bin\mkdir.exe'
```

Comment:
@mariangemarcano if I don't do it - it will invoke a Windows version of mkdir, which does not support nested dirs creation. Maybe it would work ok a katana build agent, but it will definitely fail on my local build agent.